plugins/modules/cloud/scaleway/scaleway_private_network.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Scaleway VPC management module
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: scaleway_private_network
+short_description: Scaleway private network management module
+version_added: 4.4.0
+author: Pascal MANGIN (@pastral)
+description:
+    - This module manages private network on Scaleway account
+      (U(https://developer.scaleway.com)).
+extends_documentation_fragment:
+- community.general.scaleway
+
+
+options:
+  state:
+    type: str
+    description:
+     - Indicate desired state of the VPC.
+    default: present
+    choices:
+      - present
+      - absent
+
+  project:
+    type: str
+    description:
+      - Project identifier.
+    required: true
+
+  region:
+    type: str
+    description:
+     - Scaleway region to use (for example C(par1)).
+    required: true
+    choices:
+      - ams1
+      - EMEA-NL-EVS
+      - par1
+      - EMEA-FR-PAR1
+      - par2
+      - EMEA-FR-PAR2
+      - waw1
+      - EMEA-PL-WAW1
+
+  name:
+    type: str
+    description:
+    - Name of the VPC.
+
+  tags:
+    type: list
+    elements: str
+    description:
+    - List of tags to apply to the instance.
+    default: []
+
+'''
+
+EXAMPLES = '''
+- name: Create an private network
+  community.general.scaleway_vpc:
+    project: '{{ scw_project }}'
+    name: 'vpc_one'
+    state: present
+    region: par1
+  register: vpc_creation_task
+
+- name: Make sure private network with name 'foo' is deleted in region par1
+  community.general.scaleway_vpc:
+    name: 'foo'
+    state: absent
+    region: par1
+'''
+
+RETURN = '''
+data:
```

Comment:
The return value is called `scaleway_private_network`, not `data` (see line 213).
```suggestion
scaleway_private_network:
```